temporal-testing-junit4/src/main/java/io/temporal/testing/TestWorkflowRule.java

```diff
@@ -346,4 +346,9 @@ public History getWorkflowExecutionHistory(WorkflowExecution execution) {
             .build();
     return this.blockingStub().getWorkflowExecutionHistory(request).getHistory();
   }
+
+  /** Returns the worker */
```

Comment:
Improve the comment to state which worker as it is possible to have more than one worker per rule.